examples/rails_example/app/controllers/home_controller.rb

```diff
@@ -0,0 +1,10 @@
+class HomeController < ApplicationController
+    def index
+        @flag_globally_enabled = Rails.configuration.fh_config.new_context.build.feature(:demo_feature).flag 
+        @flag_enabled_for_user_key = Rails.configuration.fh_config.new_context.user_key("some_key")
+            .build.feature(:demo_feature).flag
+        @flag_enabled_for_custom_splitting_rule = Rails.configuration.fh_config
+            .new_context.attribute_value('location_id', [5])
+            .build.feature(:demo_feature).flag
+    end
+end
```

Comment:
This is the actual example usage of the client; relevant to the review!

Reply:
I was sort of thinking the best way to do this in Rails would be to add a Rack middleware that would create the context and insert it into the request. Then you could just go request.context, and/or create a shortcut to feature, so request.feature(:demo_feature).enabled? Its not that this isn't right, I was thinking of a way to be more Railsy... What do you think?